### ISSUE TITLE
Antal enheder på ejendom

### ISSUE THREAD

**Issue by Ellebjerg** [Septima/OneDoor#39]
Vores affaldsfolk har brug for en let måde at se antal enheder på en ejendom, gerne opgjort på bolig, erhverv... Det ville være super, hvis det kunne vises på ejendommen. Ellers skal de ind på hvert enkelt husnummer for at finde tallet. Det er noget omstændeligt i større bebyggelser.

P.t. kan de finde antal boligenheder med BBR-analyse i SpS, men det ville være en del hurtigere, hvis de kunne slå det op.
 
Tilsvarende information på bygningsniveau ville også give mening. Her er p.t. kun opgørelse på areal, ikke på antal enheder.

Vh Lars

**Comment by rastermanden** [comment]
Har du et godt test-ejenmdomsnummer ?

**Comment by Ellebjerg** [comment]
1670030033 er et bud. Der er mange opgange med mange enheder. Jeg er dog i tvivl, om der er erhverv

**Comment by Ellebjerg** [comment]
Du kan også prøve med 1670002951. Her er der også erhvervsenheder.

**Comment by rastermanden** [comment]
Man kan allerede se enhederne som f.eks. her:

https://onedoor.test.septima.dk/#/septima/standard/Dawa/adresse/0a3f507a-6fa1-32b8-e044-0003ba298018/omhusnummer

Er det noget i den retning bare på en ejendom ?

**Comment by Ellebjerg** [comment]
Det er vist ikke helt det, de har brug for. På husnummeret vises hver enkelt BBR-enhed for sig som et link til, hvor man kan se mere. Det giver god mening også at vise det sådan på ejendomme, men det, som vores affaldsfolk har brug for, er en sammentælling af antal opgjort på hhv. bolig og erhverv. På den visning, som du linker til, vil de selv skulle tælle sammen, hvilket ikke rigtigt holder på en ejendom med fx 20 opgange med hver 3-4 etager.
Jeg tænker, at en visning svarende til den med BBR-enheder på husnummer er fin, men at der ud for overskriften "BBR-enheder" samtidig kunne stå "(25 boligenheder, 6 erhvervsenheder)" eller noget i den stil.

**Comment by rastermanden** [comment]
Har du nogle screenshots af den visning I ønsker?

**Comment by Ellebjerg** [comment]
På 'Om husnr' vises 'Boliger på husnummeret'. Det er en tilsvarende sammentælling på ejendomsniveau, men opfattende både boliger og erhvervsenheder, som efterspørges. Fx kunne der under 'Bygninger på fremmed grund' og 'Ejerlejligheder' være to felter med hhv. 'Boligenheder på ejendommen' og 'Erhvervsenheder på ejendommen'.

![Image](https://github.com/user-attachments/assets/380e0e92-4439-48f0-aba7-e588eee137c3)

![Image](https://github.com/user-attachments/assets/de97d132-9fa7-4590-aeb2-c2be0507c39b)

**Comment by rastermanden** [comment]
Jeg tror vi skal skelne mellem:

På husnummer
På Ejendom

**Comment by Ellebjerg** [comment]
Helt enig. Screendump fra husnummer er med for at vise, hvad det er vi efterspørger. Men det er opgørelse på ejendom, vi er interesserede i.

**Comment by rastermanden** [comment]
Er det kun for **ejerlejlighed** og **bygning på fremmede grund** ?

**Comment by Ellebjerg** [comment]
Nej, det har ikke noget med BPFG eller Ejerlejligheder at gøre. Det er bare en idé til, hvor på siden info kan placeres.
Det er alene en opgørelse af antal boligenheder og antal erhvervsenheder på ejendommen, vi er interesseret i.

**Comment by rastermanden** [comment]
Ok

Det er en smule kompliceret at få det beregnet. 

Alle enhederne skal findes gennem grund og bygninger tilhørende ejendommen

Vi vil prøve at estimere omfanget afopgaven

**Comment by Ellebjerg** [comment]
3-5 dage var godt nok ikke så lidt. Det vil i hvert fald kræve, at andre vil medfinansiere.

Jeg vil se på, om LIFA har løst det i LOIS, for så kan der måske findes en genvej ad den vej.

**Comment by DocDemar** [comment]
OK - vi sætter sagen på standby

**Comment by Ellebjerg** [comment]
Helt ok
